src/middleware.ts

```diff
@@ -0,0 +1,20 @@
+import { NextRequest, NextResponse } from 'next/server';
+
+export async function middleware(request: NextRequest) {
+  const currentUser = request.cookies.get('auth-user')?.value;
+
+  if (!currentUser && request.nextUrl.pathname.startsWith('/')) {
```

Comment:
does this check mean it is inner routing (not API request)?
request.nextUrl.pathname.startsWith('/')

Reply:
yes, inner one